cmd/podman/main_local.go

```diff
@@ -140,7 +140,7 @@ func setupRootless(cmd *cobra.Command, args []string) error {
 		became, ret, err := rootless.TryJoinFromFilePaths("", false, []string{pausePidPath})
 		if err != nil {
 			logrus.Errorf("cannot join pause process.  You may need to remove %s and stop all containers", pausePidPath)
-			logrus.Errorf("you can use `system migrate` to recreate the pause process")
+			logrus.Errorf("you can use `%s system migrate` to recreate the pause process", os.Args[0])
```

Comment:
is this happy if Args == nil?

Reply:
I don't believe that is possible.  This is basically argv[0] in main(), it points to the executable file.

Reply:
OK, I just played with it a bit this morn and couldn't break it, thx for the 411.